### DISCUSSION TITLE
Disable scrolling of the common container шт Off-canvas

### DISCUSSION THREAD

**Discussion by thefubon** [alpinejs/alpine#938] [4. General]
Can you please tell me how to disable scrolling of the common container when the side menu is open? Double scrolling appears and is inconvenient in the mobile

https://play.tailwindcss.com/2OSS4TmXXt

 Example as in Uikit. When the menu is open. The side menu will scroll and the body itself will not.

https://getuikit.com/docs/offcanvas#overlay

**Comment by KevinBatdorf** [comment]
Try this one: https://play.tailwindcss.com/yc5S82ESNq

I have a similar demo here you can check out too: https://codepen.io/KevinBatdorf/pen/ZEQJZMe?editors=1010

**Reply by thefubon** [reply]
COOL

**Reply by thefubon** [reply]
Thank you! Been looking for a solution
